tap-gui/plugins/runtime-resource-visibility.hbs.md

```diff
@@ -12,6 +12,8 @@ Do one of the following actions to access the Runtime Resources Visibility plug-
 - [Install the Tanzu Application Platform Full or View profile](../../install-intro.md)
 - [Install Tanzu Application Platform without using a profile](../../install-intro.md) and then
   install [Tanzu Application Platform GUI separately](../install-tap-gui.md)
+- If you're not using any metric server on your cluster, make sure to set the
```

Comment:
Recommendation: Either of these:

- set the `skipMetricsLookup` field to `true`
- set `skipMetricsLookup` to `true`

Reply:
Addressed, can you please re-review it? @menehune23

Reply:
Addressed, please consider resolved.